src/widget/wsearchlineedit.cpp

```diff
@@ -207,6 +208,8 @@ void WSearchLineEdit::setup(const QDomNode& node, const SkinContext& context) {
             tr("Shortcuts") + ": \n" +
             tr("Ctrl+F") + "  " +
             tr("Focus", "Give search bar input focus") + "\n" +
+            tr("Return") + " " +
+            tr("Trigger search, next press jumps to tracks view") + "\n" +
```

Comment:
Is this still correct? In my test the search is instantly applied.
And return immediately moves to the Tracks table.

Reply:
Ah, so it is the long debounce time that eats the first return.
How about this: 
```suggestion
            tr("Trigger search before search-as-you-type timeout or jumps to tracks view afterwards") + "\n" +
```

Reply:
Did you consider this comment?

Reply:
Oh, added it now. Thanks!